.circleci/config.yml

```diff
@@ -189,7 +189,10 @@ jobs:
           name: Run Docker build
           working_directory: ~/project/solidity/dashboard
           command: |
-            docker build --tag keep-dapp-token-dashboard .
+            docker build \
+            --build-arg ETH_RPC_URL=$ETH_RPC_URL \
```

Comment:
Do we need RPC or WS? I thought we want to pass `wss` endpoint address, in such case this argument should be named `ETH_WS_URL`.